bakery/service/bakery.go

```diff
@@ -76,6 +72,38 @@ func InitConfiguration(configFile, rootDir string) error {
 	return nil
 }
 
+func updateConfigProperties() {
+	// update config properties with the latest versions of available images
+	// versions are determined from the git tags
+	// this is especially useful when for the first time new child image is about to be build (without being triggered by a parent build)
+	// and there is no PARENT_VERSION property defined in the config file
+	// in such case it will be determined dynamically from the git tags and and may be used in the child Dockerfile.template
+	config.UpdateVersionProperties(versions)
+
+	// update the rest of config properties that stays the same for the duration of docker-bakery execution
+	name := getValue(GetGitUserName, "git user.name")
```

Comment:
seems that `GetGitUserName` and `GetGitUserEmail` could be a (parametrized) type, this way we won't duplicated `git user.name` and maybe `getValue` would be encapsulated in it.. wdyt?

Reply:
Actually we can get rid of this message entirely.